kubespawner/__init__.py

```diff
@@ -12,6 +12,6 @@
 # Users can simply import kubespawner.KubeSpawner in their applications
 # instead of the more verbose import kubespawner.spawner.KubeSpawner.
 
-from kubespawner.spawner import KubeSpawner
+from kubespawner.spawner import KubeSpawner, __version__
 
-__all__ = [KubeSpawner]
+__all__ = [KubeSpawner, __version__]
```

Comment:
By adding something to `__all__` we mark it as something that should be imported when you do `from kubespawner import *`. I'm not a fan of adding `__version__` because it is considered an anti-pattern to `from kubespawner import *` and I can't think of when you'd want to then have a `__version__` appear in your namespace. Especially in cases where people `from blah import *` multiple times it will lead to a lot of confusion. So having to write
```python
import kubespawner
print("k8s spawner version:", kubespawner.__version__)
```
seems like a good thing to make people do.

Reply:
Aha! Thanks for helping me understand this, I didn't know!